modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -5,40 +5,81 @@
 // * installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc
 
 [id="installation-vsphere-installer-infra-requirements_{context}"]
-= vCenter requirements
+= Requirements for vCenter cluster install
 
-Before you install an {product-title} cluster on your vCenter that uses infrastructure that the installer provisions, you must prepare your environment.
+Before you install an {product-title} cluster on your vCenter that uses installer-provisioned infrastructure, you must prepare your environment. Logging in to a user account or group that has administrative privileges is the simplest way to access all of the necessary permissions in vCenter.
+
+By configuring roles and permissions, vCenter authentication determines how a group or user can access vSphere components and objects, both globally and locally.
 
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-account_{context}"]
-== Required vCenter account privileges
-
-To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources. Using an account that has administrative privileges is the simplest way to access all of the necessary permissions.
-
-A user requires the following privileges to install an {product-title} cluster:
-
-* Datastore
-** *Allocate space*
-** *Browse datastore*
-** *Low level file operations*
-** *Remove file*
-* Folder
-** *Create folder*
-** *Delete folder*
-* vSphere Tagging
-** All privileges
-* Network
-** *Assign network*
-* Resource
-** *Assign virtual machine to resource pool*
-* Profile-driven storage
-** All privileges
-* vApp
-** All privileges
-* Virtual machine
-** All privileges
-
-For more information about creating an account with only the required privileges, see link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.security.doc/GUID-5372F580-5C23-4E9C-8A4E-EF1B4DD9033E.html[vSphere Permissions and User Management Tasks] in the vSphere documentation.
+== Applying vCenter account privileges
+
+To install an {product-title} cluster in a vCenter, the installation program requires access to an account with privileges to read and create the required resources.
+
+[IMPORTANT]
+====
+It is recommended that you apply role access privileges globally to ensure that the required permissions are met. If these permissions are not applied globally, you must determine which entities to grant the role permissions to. At minimum, you must add `read-only` access to the vCenter.
+====
+
+.Procedure
+. As a vSphere administrator, create a new custom role in your vCenter instance if you have not already done so.
++
+The new role must allow the user to read properties in both the vCenter and in all the necessary vSphere entitites.
+
+. As a vSphere administrator, apply the newly created `read-only` role to the {product-title} user.
```

Comment:
newly created "read-only" role? That role already exists..

folks - there are two levels of permissions needed for OCP user that access vsphere API

1. The user must have privileges documented in "Alternatively" section below. i.e - he should have datastore, folder permissions as we document below.
2. In addition to that (and that is not an alternative) - same user must have `Read-Only` access to node VM and all its parent entities (which basically means - `vCenter`, `Datacenter`, `Host`, `Datastore` etc)

Both of these permissions are necessary and they are not optional.

Reply:
OK, let's try this again. 😅  (Just in case, I saved a backup of the 20 Nov version, wherein I had removed the smaller table.)

I revamped the procedure with some Slack help from @gnufied. This link was helpful today, and I probably should have paid it more attention in the beginning: https://docs.pivotal.io/pks/1-4/vsphere-prepare-env.html

Are we getting closer?

Reply:
OK, let's try this again. 😅 (Just in case, I saved a backup of the 20 Nov version, wherein I had removed the smaller table.)

I revamped the procedure with some Slack help from @gnufied. This link was helpful today, and I probably should have paid it more attention in the beginning: https://docs.pivotal.io/pks/1-4/vsphere-prepare-env.html

Are we getting closer?

Reply:
This does look clear to me now. Sorry for the confusion--I was under a mistaken assumption about the custom role. Thanks for clearing it up @gnufied I will update installer docs accordingly.